### PR TITLE
Farewell custom component

### PR DESCRIPTION
**Pull request by marvin-w**
## Description

Remove custom home assistant component as its unmaintained and unneeded now. Use the core integration.

Also cleanup some setup stuff, move changelog to documentation page and remove Makefile.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist:

- [x] I have performed a self-review of my own code
- [x] The documentation has been adjusted accordingly
- [x] The changes generate no new warnings
- [x] Tests have been added that prove the fix is effective or that the feature works
- [x] The changes are documented in the changelog
- [x] The Homeassistant plugin has been adjusted in case of new config options

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ assignees: ''
 **Description of problem:**
 
 - [ ] using xknx standalone
-- [ ] using Home-Assistant knx plugin
+- [ ] using Home-Assistant knx plugin (please use the Home Assistant Issue Tracker)
```
Comment (farmio) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
- [ ] using Home-Assistant KNX integration (please use the Home Assistant Issue Tracker)
```

Comment (farmio) [reply .github/ISSUE_TEMPLATE/bug_report.md:13]: It's fine for me if a HA KNX bug is filed at xknx. Hard to find something at HAs issues again 🙃
If you prefer to have it at HA then just keep the note.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -30,5 +30,4 @@ that your change is to big and it is suggested to split it into several smaller
 - [ ] The documentation has been adjusted accordingly
 - [ ] The changes generate no new warnings
```
Comment (farmio) [inline .github/pull_request_template.md:31]: I always find this line awkward. If there has to be something to warn about, we should probably do it.
Same for the "self review" line. If one has not looked at the published code, this line will not change much about that

File: docs/_config.yml [hunk 3]
```diff
@@ -5,9 +5,9 @@ meta:
     title: XKNX - A Python KNX library
     keywords: KNX,KNX/IP,EIB,Home Automation,Home Assistant
     description: >
-        XKNX is a KNX library in Python. Helps you to control KNX devices
+        XKNX is a KNX library written in Python. Helps you to control KNX devices
```
Comment (farmio) [inline docs/_config.yml:8]:
```suggestion
        XKNX is a KNX library written in Python. It lets you to control KNX devices
```

File: docs/_config.yml [hunk 4]
```diff
@@ -5,9 +5,9 @@ meta:
...
         like Lights, Shutters, Covers, Switches, Outlets, Thermostats
-        via python scripts.
+        via python scripts. Official Home Assistant KNX library.
```
Comment (farmio) [inline docs/_config.yml:10]:
```suggestion
        via python scripts. Driver for the Home Assistant KNX integration.
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (marvin-w) [reply .github/ISSUE_TEMPLATE/bug_report.md]: done

File: .github/pull_request_template.md (hunk 2 above)
Comment (marvin-w) [reply .github/pull_request_template.md]: removed it

File: docs/home_assistant.md [hunk 5]
```diff
@@ -11,34 +11,13 @@ Home Assistant KNX Integration
...
-ln -s ~/xknx/home-assistant-plugin/custom_components/xknx ~/.homeassistant/custom_components
-```
-
-Run HA as usual either via service or by directly typing in `hass`.
 
 Running HA with local XKNX library
 ------------------------------------
 
-When running HA with the KNX integrated component once, HA will automatically install a `xknx` library version within `[hass-dependency-directory]/lib/python[python-version]/site-packages` via pip. This very often causes the problem, that the manually checked out `xknx` library is not in sync with the `xknx` library version HA already contains and uses by default. But getting both in sync is easy:
+When running HA with the KNX integrated component once, HA will automatically install a `xknx` library version within `[hass-dependency-directory]/lib/python[python-version]/site-packages` via pip. In order to test new features before a release you can run with a local xknx installation as follows:
```
Comment (farmio) [inline docs/home_assistant.md:18]:
```suggestion
When running HA with the KNX integrated component once, HA will automatically install a `xknx` library version within `[hass-dependency-directory]/lib/python[python-version]/site-packages` via pip. In order to test new features before a release you can run HA with a local xknx installation as follows:
```

File: docs/home_assistant.md [hunk 6]
```diff
@@ -11,34 +11,13 @@ Home Assistant KNX Integration
...
 
-Delete the automatically installed version:
+Delete the automatically installed version of the library:
```
Comment (farmio) [inline docs/home_assistant.md:20]: thats not needed. I never did this 🤣

**Review by farmio — APPROVED** [review]
🚀

**Pull request merged**